.github/container/test-maxtext.sh

```diff
@@ -35,7 +35,7 @@ if [[ $? -ne 0 ]]; then
 fi
 
 # Default arguments
-MULTIPROCESS=false
+HARDWARE='tpu'
```

Comment:
Do you mind leaving a comment that this is essentially a no-op? As it stands it kind of looks like we test tpu by default

Reply:
Found another way around and also updated files @terrykong